e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -129,6 +131,8 @@ describe("scenarios > search", () => {
         "eq",
         `/question/${ORDERS_QUESTION_ID}-orders`,
       );
+
+      cy.get("@search.all").should("have.length", 1);
```

Comment:
Should we add this to the other tests?

Reply:
I think it's safe enough to keep it only here.
It's great you added it, btw. We'll have an explicit fail mechanism in the future thanks to this.